sql/plan/indexed_table_access.go

```diff
@@ -138,7 +138,11 @@ func (i *IndexedTableAccess) getLookup(ctx *sql.Context, row sql.Row) (sql.Index
 }
 
 func (i *IndexedTableAccess) String() string {
-	return fmt.Sprintf("IndexedTableAccess(%s on %s)", i.Name(), formatIndexDecoratorString(i.index))
+	var filters string
+	if i.lookup != nil {
+		filters = fmt.Sprintf(" with ranges: %s", i.lookup.Ranges().DebugString())
```

Comment:
Ranges.String()?

Reply:
I should have renamed it `HumanReadableString()`